config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
@jaronoff97 👍 or 👎 for this autogenned file

Reply:
yeah remove this change :)